src/Bundle/Maker/MakeFactoryData.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
Let's remove this (not part of this project's current cs).

Reply:
oops this is in my phpstorm template :smile: